test/modules/http/intercept/http.request.test.ts

```diff
@@ -229,3 +229,27 @@ it('intercepts an http.request given RequestOptions without a protocol', async (
 
   expect(requestId).toMatch(UUID_REGEXP)
 })
+
+it('intercepts an http.request with three argument form: URL, options, and callback', async () => {
```

Comment:
Would be also great to add that ambiguous scenario when both URL and options include `path` (the one we talked about in the comments). Just so we reflect that behavior correctly as well.

Reply:
done